packages/builder/src/pages/builder/app/[application]/data/datasource/[selectedDatasource]/rest/[query]/index.svelte

```diff
@@ -118,7 +118,12 @@
   function checkQueryName(inputUrl = null) {
     if (query && (!query.name || query.flags.urlName)) {
       query.flags.urlName = true
-      query.name = url || inputUrl
+      query.name =
```

Comment:
NAB: I'd maybe just pull out this replacement into a little function.

`const cleanUrl = str => url?.replace(/(http)|(https)|[{}:]/g, "")?.replaceAll(".", "_")?.replaceAll("/", " ")?.trim()`